beta-src/src/components/ui/WDCommandButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { Stack } from "@mui/material";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+import WDButton from "./WDButton";
+import VoteType from "../../types/Vote";
+import Vote from "../../enums/Vote";
+
+interface voteProps {
+  voteState: VoteType;
+  toggleVote: (vote: Vote) => void;
+}
+
+const WDCommandButtons: React.FC<voteProps> = function ({
+  voteState,
+  toggleVote,
+}): React.ReactElement {
+  const { draw, pause, cancel } = voteState;
+
+  return (
+    <Stack direction="row" spacing={2} alignItems="center">
+      <WDButton
+        color={draw ? "secondary" : "primary"}
+        onClick={() => toggleVote(Vote.DRAW)}
+        startIcon={draw ? <WDCheckmarkIcon /> : ""}
+      >
+        Draw
+      </WDButton>
+      <WDButton
+        color={pause ? "secondary" : "primary"}
+        onClick={() => toggleVote(Vote.PAUSE)}
+        startIcon={pause ? <WDCheckmarkIcon /> : ""}
+      >
+        Pause
+      </WDButton>
+      <WDButton
+        color={cancel ? "secondary" : "primary"}
+        onClick={() => toggleVote(Vote.CANCEL)}
+        startIcon={cancel ? <WDCheckmarkIcon /> : ""}
+      >
+        Cancel
+      </WDButton>
```

Comment:
As mentioned by @hyukcodazen we should do a map here. You should be able to loop through the VoteEnum keys and use those.